manifests/download.pp

```diff
@@ -191,6 +191,8 @@
     'i386' : { $arch = 'i586' }
     'x86_64' : { $arch = 'x64' }
     'amd64' : { $arch = 'x64' }
+    'aarch64' : { $arch = 'aarch64' }
+    'amd64' : { $arch = 'aarch64' }
```

Comment:
I notice there are two `amd64` keys in the hash after your change. I think this would cause problems. Should the key name be something different?